gunicorn/workers/workertmp.py

```diff
@@ -35,14 +36,12 @@ def __init__(self, cfg):
             os.close(fd)
             raise
 
-        self.spinner = 0
-
     def notify(self):
-        self.spinner = (self.spinner + 1) % 2
-        os.fchmod(self._tmp.fileno(), self.spinner)
+        new_time = time.monotonic()
+        os.utime(self._tmp.name, (new_time, new_time))
 
     def last_update(self):
-        return os.fstat(self._tmp.fileno()).st_ctime
+        return os.fstat(self._tmp.fileno()).st_mtime
```

Comment:
why the change? `st_ctime` is platform dependent where `st_mtime` will always tell the most recent content modification expressed in seconds.

Reply:
https://docs.python.org/3/library/os.html?highlight=st_ctime#os.stat_result.st_ctime

Reply:
In fact, my idea is to use the member variable to store the time without modifying the file

Reply:
@skytoup  what do you mean? `os.fstat` is atomic, using the modification time is actually very safe so i'm not sure we would like to change it. Such change can be part of another ticket but let's focus on making the time monotonic imo.

Reply:
This object is shared between the master process and the child process at fork, both with access to the file descriptor of an open file. After the fork, the memory is not shared. That's why we use the filesystem modification time—to signal between processes.

Reply:
And we need ctime instead of mtime because we only change the metadata (permissions), and not the content.

Reply:
`ctime` is not portable this is why I was using `mtime`. I would prefer to keep it that way.

Reply:
I'm confused, @benoitc. This change is from `ctime` to `mtime` and you're asking "why the change" but then you say that you prefer `mtime`?

Reply:
Even if `ctime` is platform dependent, I think we _have_ to use it because we don't modify the file content in any way. We only modify the permissions.

Reply:
I'm glad you caught the change. I missed it! @skytoup would you mind changing this back to `ctime`, please?

Reply:
Oh, wait! I am wrong. I got confused.

We are changing this to use utime instead of fchmod, so it is the modified time that changes. In that case, I think the PR is right.